app/services/force/lease_up_service.rb

```diff
@@ -12,25 +12,28 @@ def lease_up_listings
       )))
     end
 
-    def lease_up_listing_applications(listing_id)
-      # TO DO: Temp fix limit to 500, will paginate in #159530254
-      application_data = massage(query(%(
-        SELECT #{query_fields(:lease_up_listing_application)}
-        FROM Application_Preference__c
-        WHERE Listing_Preference_ID__c IN
-        (SELECT Id FROM Listing_Lottery_Preference__c WHERE Listing__c = '#{listing_id}')
-        AND Preference_Lottery_Rank__c != NULL
-        ORDER BY Preference_Order__c, Preference_Lottery_Rank__c
-        ASC NULLS LAST LIMIT 500
-      )))
+    def lease_up_listing_applications(listing_id, opts = { page: 0 })
```

Comment:
i'm also curious if we need to add any sort of sanitization here to avoid soql injection

Reply:
We probably should. We can capture this in different story

Reply:
https://www.pivotaltracker.com/story/show/160629818